app/templates/suppliers/edit_company_registration_number.html

```diff
@@ -27,9 +27,7 @@
 <div class="single-question-page">
   <div class="grid-row">
     <div class="column-two-thirds">
-      {% with heading = "Are you registered with Companies House?", smaller = True %}
-        {% include "toolkit/page-heading.html" %}
-      {% endwith %}
+      <h1 class="govuk-heading-l">$1</h1>
```

Comment:
Should this be `"Are you registered with Companies House?"`

Reply:
Yeah, this looks like a regex issue (same for the others)

Reply:
```suggestion
      <h1 class="govuk-heading-l">Are you registered with Companies House?</h1>
```

Reply:
Oof whoops